test/JDBC/expected/col_length-vu-prepare.out

```diff
@@ -51,11 +51,78 @@ CREATE TABLE sys_col_length_test_schema.test_table(
 );
 GO
 
-INSERT INTO sys_col_length_test_schema.test_table (col_char, col_varchar, col_varbinary)
-VALUES ('ABCDEF', 'Hello, World!', 0x0123456789ABCDEF)
+-- for user defined data types
+-- Create User-Defined Types
+CREATE TYPE custom_char_10 FROM CHAR(10);
```

Comment:
It's better to add a `GO` after each statement.
```
CREATE TYPE custom_char_10 FROM CHAR(10);
GO
```
It would make sure that one statement failure doesn't stop other statements from execution.